scripts/serve-dev-wsl.sh

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env bash
```

Comment:
We don't generally accept `scripts` since they are used as part of the build process only